docs/deploying.md

```diff
@@ -0,0 +1,209 @@
+# Deploying a Mesos Cluster with Calico.
```

Comment:
If we are going to have multiple guides, it may be good to distinguish this guide from others. 

Something like `Deploying a Dockerized Mesos Cluster with Calico`